create-doks-with-terraform-flux/variables.tf

```diff
@@ -0,0 +1,76 @@
+# DOKS
+variable "do_token" {
+  type = string
+  description = "DigitalOcean API token"
+  sensitive = true
+}
+
+variable "doks_cluster_name" {
+  type = string
+  default = "doks-fluxcd-cluster"
+  description = "DOKS cluster name"
+}
+
+variable "doks_cluster_region" {
+  type = string
+  default = "lon1"
```

Comment:
Should we recommend that people change this value to a region that close to them? Not required,  but it might help the cluster spin up more quickly.

Reply:
Thanks for the suggestion. Fixed in the latest commit.